src/policy/fees.cpp

```diff
@@ -195,6 +202,56 @@ TxConfirmStats::TxConfirmStats(const std::vector<double>& defaultBuckets,
     resizeInMemoryCounters(buckets.size());
 }
 
+TxConfirmStats::TxConfirmStats(const std::vector<double>& defaultBuckets, const std::map<double, unsigned int>& defaultBucketMap, const TxConfirmStats& txstatsSource)
+    : buckets(defaultBuckets), bucketMap(defaultBucketMap),
+      decay(txstatsSource.decay), scale(txstatsSource.scale)
+{
+    const unsigned int maxPeriods = txstatsSource.confAvg.size();
```

Comment:
Nit: Make `maxPeriods` – and the three following `i` – `size_t` instead to avoid implicit conversions?